TYPO3.Media/Classes/TYPO3/Media/Domain/Strategy/AbstractAssetUsageStrategy.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace TYPO3\Media\Domain\Strategy;
+
+/*
+ * This file is part of the TYPO3.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Persistence\PersistenceManagerInterface;
+use TYPO3\Flow\Utility\TypeHandling;
+use TYPO3\Media\Domain\Model\AssetInterface;
+use TYPO3\Media\Domain\Model\Dto\UsageReference;
+use TYPO3\Media\Domain\Model\Image;
+use TYPO3\Media\Domain\Strategy\AssetUsageStrategyInterface;
+use TYPO3\Neos\Domain\Repository\SiteRepository;
+use TYPO3\Neos\Service\UserService;
+use TYPO3\Eel\FlowQuery\FlowQuery;
+use TYPO3\Neos\Controller\CreateContentContextTrait;
+use TYPO3\TYPO3CR\Domain\Factory\NodeFactory;
+use TYPO3\TYPO3CR\Domain\Repository\NodeDataRepository;
+use TYPO3\Neos\Domain\Service\UserService as DomainUserService;
+
```

Comment:
Lots of unused imports here
![image](https://cloud.githubusercontent.com/assets/95582/14703998/f2cb22aa-07b2-11e6-815d-4bd143b90712.png)